src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp

```diff
@@ -124,6 +124,7 @@ class PolicyManager final : public Singleton<PolicyManager>
                 Utils::parseStrToTime(newPolicy.at("vulnerability-detection").at("feed-update-interval"));
             newPolicy["updater"]["ondemand"] = false;
             newPolicy["updater"]["topicName"] = "vulnerability_feed_manager";
+            newPolicy["updater"]["translationsTopic"] = "vulnerability_feed_manager_translations";
```

Comment:
The router's subscriberId argument could be in a constant. So different subscribers in this class could use the same constant.
In this case the translationTopic don't match with the one used in databaseFeedManager.hpp

Reply:
Here we are setting the default configuration, and assigning `vulnerability_feed_manager_translations` as a `topicName`.

In ` databaseFeedManager.hpp` we are using this same configuration value `["updater"]["translationsTopic"]`.
The `topicName` and the `subscriberId` aren't required to be the same.

Reply:
You are right. My bad